contentcuration/contentcuration/frontend/shared/data/index.js

```diff
@@ -35,6 +36,9 @@ export function setupSchema() {
     [CHANGES_TABLE]: 'rev++,[table+key]',
     // A special table for keeping track of change locks
     [CHANGE_LOCKS_TABLE]: 'id++,tracker_id,expiry',
+    // A special table for keeping a current user session
+    // (it contains only a single row)
+    [CURRENT_SESSION_TABLE]: '',
```

Comment:
I assume that there is no need to increase the database version on schema updates while still on `develop`, right?

Reply:
No, not currently - and unfortunately, we would need to add some additional logic to handle version upgrades.